kie-wb-common-react/boxed-expression-editor/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "boxed-expression-editor",
+  "main": "server.js",
+  "private": true,
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "webpack --config webpack.prod.js",
+    "start": "webpack serve --hot --color --progress --config webpack.dev.js",
+    "start:prod": "node server.js",
+    "test": "jest",
+    "lint": "eslint --ext .tsx,.js ./src/",
+    "type-check": "tsc --noEmit",
+    "quality-checks": "npm run type-check && npm run lint && npm run test"
+  },
+  "devDependencies": {
+    "@testing-library/jest-dom": "^5.11.5",
+    "@types/enzyme": "^3.10.7",
+    "@types/enzyme-adapter-react-16": "^1.0.6",
+    "@types/jest": "^26.0.14",
+    "@types/react-dom": "^16.9.9",
+    "@types/react-router-dom": "^5.1.6",
+    "@types/victory": "^33.1.4",
+    "@typescript-eslint/eslint-plugin": "^4.4.1",
+    "@typescript-eslint/parser": "^4.4.1",
+    "copy-webpack-plugin": "^6.2.1",
+    "css-loader": "^5.0.0",
+    "dotenv-webpack": "^4.0.0",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.5",
+    "enzyme-to-json": "^3.6.1",
+    "eslint": "^7.11.0",
+    "eslint-plugin-react": "^7.21.4",
+    "eslint-plugin-react-hooks": "^4.1.2",
+    "express": "^4.17.1",
+    "file-loader": "^6.1.1",
+    "html-webpack-plugin": "^4.5.0",
+    "identity-obj-proxy": "^3.0.0",
+    "imagemin": "^7.0.0",
+    "jest": "^26.5.3",
+    "mini-css-extract-plugin": "^1.0.0",
+    "optimize-css-assets-webpack-plugin": "^5.0.4",
+    "prettier": "^2.1.2",
+    "prop-types": "^15.6.1",
+    "raw-loader": "^4.0.2",
+    "react-axe": "^3.5.4",
+    "react-docgen-typescript-loader": "^3.7.2",
+    "react-router-dom": "^5.2.0",
+    "regenerator-runtime": "^0.13.7",
+    "rimraf": "^3.0.2",
+    "style-loader": "^2.0.0",
+    "svg-url-loader": "^6.0.0",
+    "terser-webpack-plugin": "^5.0.0",
+    "ts-jest": "^26.4.1",
+    "ts-loader": "^8.0.5",
+    "tsconfig-paths-webpack-plugin": "^3.3.0",
+    "tslib": "^2.0.3",
+    "typescript": "^4.0.3",
+    "url-loader": "^4.1.1",
+    "webpack": "^5.1.3",
+    "webpack-cli": "^4.0.0",
+    "webpack-dev-server": "^3.11.0",
+    "webpack-merge": "^5.2.0"
+  },
+  "dependencies": {
+    "@patternfly/react-core": "^4.63.3",
+    "@patternfly/react-icons": "^4.7.11",
+    "@patternfly/react-styles": "^4.7.8",
+    "lodash": "^4.17.20",
+    "react": "16.12.0",
```

Comment:
I believe we can postpone the React as `peerDependency`.. just but a reminder ;-)

Reply:
Yes, in the KOGITO-3629 branch I've already changed dependencies to not have `react` and `react-dom` in the `dependencies` section. Instead we will have it as `peerDependencies` and `devDependencies` (for testing purpose)